docs/config-file/v2.rst

```diff
@@ -512,6 +517,43 @@ Note that 0 means *normal rank*, not *no rank*.
    Is better to decrease the rank of pages you want to deprecate,
    rather than increasing the rank of the other pages.
 
+search.ignore
+`````````````
+
+Don't index files matching a pattern.
+This is, you won't see search results from these files.
```

Comment:
Reading the code, I guess that the ignored page will affect _only_ the version that has this config file, right? If that's correct, we should probably communicate this in the documentation.

Reply:
All options from the configuration file are per-version.

Reply:
Are we mentioning that somewhere? That's my point.

Reply:
We do in https://docs.readthedocs.io/en/stable/config-file/index.html, but I can see helpful mentioning it again at the top of this file.